packages/asana/src/Adaptor.js

```diff
@@ -214,6 +214,41 @@ export function upsertTask(projectGid, params, callback) {
   };
 }
 
+/**
+ * Search for tasks in a workspace by task name.
+ * @public
+ * @example <caption>Search for a task by name</caption>
+ * searchTask("Test Search Task", {
+ *   resource_subtype: "default_task",
+ *   sort_by: "modified_at"
+ * });
+ * @function
+ * @param {string} task - The text or name of the task to search for.
+ * @param {object} [query] - Query params. See {@link https://developers.asana.com/reference/searchtasksforworkspace Docs} for a list of valid parameters.
+ * @returns {Operation} An operation that, when executed, returns the search results in state.data.
+ */
+export function searchTask(task, query = {}) {
```

Comment:
Can we add `options` with:

- `workspaceGid` (which overrides state.configuration if it's set)
- `projectId` which filters the project. I know this is probably just a query param but because most APIs use `projectId`, I want to see it represented in the signature

Reply:
I think projects.any is what we want. But ok, let's leave it out for now. Users can build their own URL